Detectors/ITSMFT/MFT/reconstruction/include/MFTReconstruction/Track.h

```diff
@@ -9,36 +9,47 @@
 // or submit itself to any jurisdiction.
 
 /// \file Track.h
-/// \brief Simple track obtained from hits
-/// \author bogdan.vulpescu@cern.ch 
-/// \date 11/10/2016
+/// \brief Definition of the MFT track
```

Comment:
We move data classes to DataFormat, e.g. https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Detectors/ITSMFT/ITS/include/DataFormatsITS/TrackITS.h
Could you please do the same for MFT track?